cs50/2017/fall/recover/check50/generate_jpg_data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# get the sizes of the jpegs in bytes
+wc -c *.jpg > sizes.txt
+
+# get the hashes of the jpegs
+md5sum *.jpg > hashes.txt
```

Comment:
The `hash` method uses SHA256, no? Has this been tested?

Reply:
yes, this works

Reply:
Actually, that's a good point. How does this work if check50's `hash` function indeed uses SHA256? Cf. https://github.com/cs50/check50/blob/develop/check50.py#L621. It's handy being able to use `md5sum` at the command line (e.g., to have students tell us their output's hash), so we should perhaps change `hash()` to use MD5 too? 

CC @brianyu28

Reply:
Easy change to make if we want it, but we'll have to be careful to simultaneously update all existing checks that use SHA-256 hashes to now use MD5 instead.